src/libertem/io/dataset/raw.py

```diff
@@ -95,10 +95,27 @@ class RawFileDataSet(DataSet):
     dtype: numpy dtype
         The dtype of the data as it is on disk. Can contain endian indicator, for
         example >u2 for big-endian 16bit data.
+
+    num_partitions: int, optional
+        Override the number of partitions. This is useful if the
+        default heuristic doesn't work well.
```

Comment:
Perhaps rather than "heuristic" we could mention why we added this option.


```suggestion
        Override the number of partitions. This is useful if the
        default number of partitions, chosen based on common workloads,
        creates partitions which are too large (or small) for the UDFs
        being run on this dataset.
```